funnel/views/project.py

```diff
@@ -72,70 +72,70 @@ class CountWords:
     CountWords(__("Be the first to register!"), '', __("Be the first follower!"), ''),
     CountWords(
         __("One registration so far"),
-        __("You are the first"),
+        __("You have registered"),
         __("One follower so far"),
-        __("You are the first"),
+        __("You are following this"),
```

Comment:
These changes have to be reflected in the test as well — the test is now broken.